README.md

```diff
@@ -66,11 +102,61 @@ helm repo add robusta https://robusta-charts.storage.googleapis.com && helm repo
 helm install robusta robusta/robusta -f ./generated_values.yaml
 ```
 
-Full instructions are in our [installation guide](https://docs.robusta.dev/master/installation.html).
+[Detailed instructions »](https://docs.robusta.dev/master/installation.html)
+
+<!-- <p align="right">(<a href="#top">back to top</a>)</p> -->
+
+## Documentation
+Interested? Learn more about Robusta
+
+* [Architecture](https://docs.robusta.dev/master/architecture.html)
+* [Upgrade and Uninstall](https://docs.robusta.dev/master/upgrade.html)
+* [Configuration Guide](https://docs.robusta.dev/master/user-guide/configuration.html)
+* [Triggers](https://docs.robusta.dev/master/catalog/triggers/index.html)
+
+[Full documentation »](https://docs.robusta.dev/master/index.html)
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Contributing
+Any contribution is most welcome, fork the project, make changes and create a PR. We can't wait to review and merge your PR's!
+
+* [Coding Conventions](https://docs.robusta.dev/master/developer-guide/platform/coding-conventions.html#formatting-and-coding-conventions)
+* [Core platform contributions](https://docs.robusta.dev/master/developer-guide/platform/dev-setup.html)
+* [Documentation contributions](https://docs.robusta.dev/master/developer-guide/platform/docs-contributions.html)
+* [Issue/Feature](https://github.com/robusta-dev/robusta/issues)
+
+## Contact
+
+* Slack - [robustacommunity.slack.com](https://join.slack.com/t/robustacommunity/shared_invite/zt-10rkepc5s-FnXKvGjrBmiTkKdrgDr~wg)
+* Twitter - [@RobustaDev](https://twitter.com/RobustaDev)
+* LinkedIn - [robusta-dev](https://www.linkedin.com/company/robusta-dev/)
+* Jobs - [jobs@robusta.dev ](mailto:jobs@robusta.dev)
+* Commercial Support - [support@robusta.dev ](mailto:support@robusta.dev )
```

Comment:
Maybe repace "Commercial Support" with "Email Support" - I want people to know that they can email us even if they're not a paying customer.

Reply:
Yes understood. Made the change